packages/react/checkbox/src/Checkbox.tsx

```diff
@@ -97,7 +97,7 @@ const Checkbox = React.forwardRef((props, forwardedRef) => {
       {isFormControl && (
         <BubbleInput
           control={button}
-          stoppedPropagation={hasConsumerStoppedPropagationRef.current}
+          bubbles={!hasConsumerStoppedPropagationRef.current}
```

Comment:
I renamed this prop again because it was bothering me that the prop name described what was happening externally. The component shouldn't care what makes it bubble or not, we just want to tell it whether to bubble.